lib/_sendEvent.ts

```diff
@@ -16,6 +17,9 @@ export type InsightsEvent = {
   filters?: string[];
 };
 
+const USER_AGENT = encodeURIComponent(
+  `Algolia insights for vanilla JavaScript (${version})`
```

Comment:
```suggestion
  `Algolia insights for JavaScript (${version})`
```

I don't think the `vanilla` is important. I've removed it from the API Client.

Reply:
done here, thanks: 1ef66dc